packages/tailwindcss/src/utilities.ts

```diff
@@ -1806,21 +1779,13 @@ export function createUtilities(theme: Theme) {
   functionalUtility('scroll-px', {
     supportsNegative: true,
     themeKeys: ['--scroll-padding', '--spacing'],
-    handle: (value) => [
-      decl('--tw-sort', 'scroll-padding-inline'),
-      decl('scroll-padding-left', value),
-      decl('scroll-padding-right', value),
-    ],
+    handle: (value) => [decl('scroll-padding-inline', value)],
   })
 
   functionalUtility('scroll-py', {
     supportsNegative: true,
     themeKeys: ['--scroll-padding', '--spacing'],
-    handle: (value) => [
-      decl('--tw-sort', 'scroll-padding-block'),
-      decl('scroll-padding-top', value),
-      decl('scroll-padding-bottom', value),
-    ],
+    handle: (value) => [decl('scroll-padding-block', value)],
   })
 
   functionalUtility('scroll-ps', {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +2029 to +2030]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Not introduced in this PR, but it looks like we already use inline start/end on `space-x` utilities. I wonder if we can further simplify this to something like:

```suggestion
        decl('margin-inline', `calc(${value} * calc(1 - var(--tw-space-x-reverse)))`),
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/tailwindlabs/tailwindcss/14805?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->